packages/storybook-addon-featureflags/src/switch.js

```diff
@@ -0,0 +1,73 @@
+import * as glamor from 'glamor'
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+
+import {layout} from '@pluralsight/ps-design-system-core'
+import { blue, utility, contrast } from '@pluralsight/ps-design-system-core/js/colors.2020.js'
+
+const styles = {
+  container: _ =>
+    glamor.css({
+      border: `1px solid ${utility[40]}`,
```

Comment:
this `utility` usage looks really weird.  Not sure what else to do, though, if the name of the color is "utility 40".

Reply:
yeah I confused this with other values in the beta colors figma project need to correct this.